### PR TITLE
Add SevenNet-MF-ompa

### PR DESCRIPTION
**Pull request by YutackPark**
Dear Devs,

Thank you for maintaining the leaderboard. We are submitting our new model, "SevenNet-MF-ompa," as our first non-compliant model. I’ve made every effort to follow the submission guidelines and am happy to address any additional requirements that would benefit the maintainers or readers.

Best regards,
Yutack Park

Please check the following items before submitting your PR:

- [x] I have created a new folder and YAML metadata file `models/<arch_name>/<model_variant>.yml` for my submission. `arch_name` is the name of the architecture and `model_variant.yml` includes things like author details, training set names and important hyperparameters.
- [x] I have added the new model as a new attribute on the [`Model.<arch_name>` enum](https://github.com/janosh/matbench-discovery/blob/57d0d0c8a14cd317/matbench_discovery/enums.py#L274) in `enums.py`.
- [x] I have uploaded the energy/force/stress model prediction file for the WBM test set to Figshare or another cloud storage service (`<yyyy-mm-dd>-<model_variant>-preds.csv.gz`).
- [x] I have uploaded the model-relaxed structures file to Figshare or another cloud storage service (`<yyyy-mm-dd>-wbm-IS2RE-FIRE.json.gz`).
- [x] I have uploaded the phonon predictions to Figshare or another cloud storage service (`<yyyy-mm-dd>-kappa-103-FIRE-<values-of-dist|fmax|symprec>.gz`).
- [x] I have included the urls to the Figshare files in the YAML metadata file (`models/<arch_name>/<model_variant>.yml`). If not using Figshare I have included the URLs to the cloud storage service in the description of the PR.
- [x] I have included the test script (`test_<arch_name>_<task>.py` for `task` in `discovery`, `kappa`, `diatomics`) that generated the prediction files.

### CODE REVIEW

File: models/sevennet/sevennet-mf-ompa.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+train_task: S2EFS
+test_task: IS2RE-SR
+targets: EFS_G
+model_type: UIP
+model_params: 25_734_966
+n_estimators: 1
+
+hyperparams:
+  max_force: 0.02
+  max_steps: 800
```
Comment (janosh) [inline models/sevennet/sevennet-mf-ompa.yml:56]: could you comment on the departure from the standard `max_steps=500` and `max_force=0.05` here? did you find these changes to make a significant difference in the metrics?

**Review by janosh — COMMENTED** [review]
thanks a lot @YutackPark! looks like a great model 👍 

same comment as in https://github.com/janosh/matbench-discovery/pull/220#pullrequestreview-2686084448, could you share the `models/sevennet/test_sevennet_kappa.py` script that generated the `κ_SRME` result?

File: models/sevennet/sevennet-mf-ompa.yml (hunk 1 above)
Comment (YutackPark) [reply models/sevennet/sevennet-mf-ompa.yml:56]: There was no significant difference overall, but the F1 score increased by approximately in the range between 0.0003 and 0.0005. I think that as the model becomes more accurate, tighter relaxation settings is slightly more beneficial for its predictions.

**Comment by janosh** [comment]
@YutackPark [in case you missed it](https://github.com/janosh/matbench-discovery/pull/221#pullrequestreview-2686839291), could you share the `models/sevennet/test_sevennet_kappa.py` script that generated the `κ_SRME` result?

File: models/sevennet/sevennet-0.yml [hunk 2]
```diff
@@ -71,6 +71,7 @@ requirements:
   numpy: 1.26.4
 
 training_set: [MPtrj]
+status: deprecated
```
Comment (CompRhys) [inline models/sevennet/sevennet-0.yml]:
```suggestion
status: superseded
```

**Comment by YutackPark** [comment]
@janosh Sorry for the late response. As we were using old https://github.com/MPA2suite/k_SRME code in evaluating the metric, I did double check with the code implemented in matbench yesterday but obtained slightly different result.

There's a high change that I have made some mistakes, especially in hyperparameters, as this is my first time computing the kappa metric myself. I'll consult with our co-worker who is expert in this metric. Thanks for the patient I'll let you know as soon as the issue is solved!

**Comment by YutackPark** [comment]
Do you prefer using single `test_sevennet_{discovery/kappa/diatomics}.py`? As we have clean interface for calculators I can easily make a single test file for range of SevenNet variants

**Comment by YutackPark** [comment]
> @janosh Sorry for the late response. As we were using old https://github.com/MPA2suite/k_SRME code in evaluating the metric, I did double check with the code implemented in matbench yesterday but obtained slightly different result.
> 
> There's a high change that I have made some mistakes, especially in hyperparameters, as this is my first time computing the kappa metric myself. I'll consult with our co-worker who is expert in this metric. Thanks for the patient I'll let you know as soon as the issue is solved!

The cause was GPU OOM error of one material. After fix, I was able reproduce the results. 
diatomic & kappa test scripts are added

**Review by janosh — APPROVED** [review]
> Do you prefer using single `test_sevennet_{discovery/kappa/diatomics}.py`?

that's perfect, thank you 👍

**Comment by janosh** [comment]
@YutackPark i completely missed that you used `displacement_distance = 0.03` in `test_sevennet_kappa.py`. in case you also ran that, could you share the prediction files for `displacement_distance = 0.01`? in fact, feel free to share all displacements you tried

https://github.com/janosh/matbench-discovery/blob/c41df43874b917e5e404161c5d41b1c24c1d0ded/models/sevennet/test_sevennet_kappa.py#L53

**Comment by janosh** [comment]
@YutackPark bump

**Comment by YutackPark** [comment]
Sorry for my super slow response! We don't have the result with 'displacement_distan ce=0.1'. But I think I can easily make one. Please wait a little longer.

**Comment by janosh** [comment]
take your time. not urgent. just thought my comment got lost
